app/jinja_filters.py

```diff
@@ -52,6 +52,29 @@ def format_percentage(value):
     return '{}%'.format(value)
 
 
+@blueprint.app_template_filter()
+def format_address_list(address1=None, address2=None):
+    """
+        This function format_address_list accepts two address', a user submitted address and a metadata address
+        If all the items in address list are 'Undefined' (nothing in answer_store) then we retrieve the metadata.
+
+        :param address1 user entered date
```

Comment:
This isn't a date is it?  Is this a copy/pasta error? ^^
(Same comment with the line below)